doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -754,6 +754,31 @@ The following attributes are supported:
 * ``affiliation`` Affiliation
 * ``filePIDsEnabled`` ("true" or "false") Restricted to use by superusers and only when the :ref:`:AllowEnablingFilePIDsPerCollection <:AllowEnablingFilePIDsPerCollection>` setting is true. Enables or disables registration of file-level PIDs in datasets within the collection (overriding the instance-wide setting).
 
+.. _collection-storage-quotas:
+  
+Collection Storage Quotas
+~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Any UI impact? Should we add to the long, dynamic list of rules? Or defer until we're off JSF? Screenshot below:

![Screenshot 2023-11-28 at 12 02 31 PM](https://github.com/IQSS/dataverse/assets/21006/147f80b9-9385-454a-840c-5a19866f5f5c)

Reply:
Yes, the remaining quota info will be shown to the user on the file upload page, above. The code for that is already in the develop branch, it was added in the proof of concept/phase one pr (#9509).